spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -181,9 +181,9 @@
               caffeinated.unassign(decaffeinated)
             end
 
-            expect(user.connected?(decaffeinated)).to be true
-            expect(user.connected?(caffeinated)).to be true
-            expect(caffeinated.connected?(decaffeinated)).to be false
+            expect(user.connected?(decaffeinated)).to be_truthy
```

Comment:
Looks like a find-replace went one step too far. There aren't too many instances of that switch , I'll ensure they all retain meaning.